app/build.gradle

```diff
@@ -148,6 +148,7 @@ dependencies {
     implementation "androidx.constraintlayout:constraintlayout:$constraintLayout"
     implementation "androidx.swiperefreshlayout:swiperefreshlayout:$swipeRefreshLayout"
     implementation "androidx.webkit:webkit:$webkit"
+    implementation "androidx.security:security-crypto:1.1.0-alpha03"
```

Comment:
question: where are we using this dependency?

Reply:
we don't use it, my bad. It is a left over of something else I was doing. Removing it